src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/excerpt/main.js

```diff
@@ -60,6 +60,8 @@ define([], function() {
 
                 this.sandbox.dom.html(this.$el, tpl);
 
+                this.sandbox.dom.removeClass('.preview-update', 'preview-update');
```

Comment:
this is used to avoid that this fields are passed to the preview system i will comment it!